Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -299,7 +299,12 @@ extension BrowserViewController: WKNavigationDelegate {
           // The tracker protection script
           // This script will track what is blocked and increase stats
           .trackerProtectionStats: requestURL.isWebPage(includeDataURIs: false) &&
-                                   domainForMainFrame.isShieldExpected(.AdblockAndTp, considerAllShieldsOption: true)
+                                   domainForMainFrame.isShieldExpected(.AdblockAndTp, considerAllShieldsOption: true),
+
+          // Add Brave search result ads processing script.
```

Comment:
Would be great to change this comment like the above comment, i.e., `// This script will ...`